src/org/labkey/targetedms/TargetedMSController.java

```diff
@@ -3440,12 +3441,83 @@ protected QueryView createQueryView(RunDetailsForm form, BindException errors, b
             TargetedMSSchema schema = new TargetedMSSchema(getUser(), getContainer());
             settings.getQueryParameters().put("RUN_ID", form._id);
             QueryView view = schema.createView(getViewContext(), settings, errors);
-            view.setShowDetailsColumn(false);
+            List<DisplayColumn> dc = view.getDisplayColumns();
             view.setShowFilterDescription(false);
             return view;
         }
     }
 
+    @RequiresPermission(ReadPermission.class)
+    public class ShowSkylineAuditLogExtraInfoAJAXAction extends SimpleViewAction<SkylineAuditLogExtraInfoForm>
+    {
+        @Override
+        public ModelAndView getView(SkylineAuditLogExtraInfoForm form, BindException errors) throws Exception
+        {
+            SkylineAuditLogExtraInfoBean bean;
+            String errMessage = null;
+            AuditLogEntry ent = AuditLogEntry.retrieve(form.getEntryId());
```

Comment:
What is the current understanding of a situation when different versions of the same document (same document GUID) are loaded into different containers? Are they still considered versions of the same document or different documents?
Because if they are versions of the same document then I cannot attribute an audit log entry to a single container.

Reply:
I imagine that different versions of a document will typically be loaded into a single container, but good point on the potential for sharing.

Instead of a simple filter on container then, maybe we'll need to get audit entry, get the list of documents that it's linked to, and verify that at least one of them is in the current container?

Reply:
Yes, that's what I am working on right now.